pandas/io/json/table_schema.py

```diff
@@ -0,0 +1,179 @@
+"""
+Table Schema builders
+
+http://specs.frictionlessdata.io/json-table-schema/
+"""
+from collections import OrderedDict
+
+from pandas.types.common import (
+    is_integer_dtype, is_timedelta64_dtype, is_numeric_dtype,
+    is_bool_dtype, is_datetime64_dtype, is_datetime64tz_dtype,
+    is_categorical_dtype, is_period_dtype, is_string_dtype
+)
+
+
+def _as_json_table_type(x):
```

Comment:
very minor, but since now explicity importing, you dont need to use private "_" leading function names. but no big deal.